lib/src/migrators/module.dart

```diff
@@ -599,12 +621,17 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
   /// Declares a variable within this stylesheet, in the current local scope if
   /// it exists, or as a global variable otherwise.
   void _declareVariable(VariableDeclaration node) {
-    var name = node.name;
     if (_scope.isGlobal || node.isGlobal) {
-      name = _unprefix(node.name);
+      var name = node.name;
+      if (name.startsWith('-') &&
+          references.referencedOutsideDeclaringStylesheet(node)) {
+        // Remove leading `-` since private members can't be accessed outside
+        // the module they're declared in.
+        name = name.substring(1);
+      }
+      name = _unprefix(name);
       if (name != node.name) {
-        addPatch(
-            patchDelete(node.span, start: 1, end: prefixToRemove.length + 1));
+        _renameMember(node, name);
```

Comment:
Style nit: make this an inline `if`. I think there are a few more you can make inline too.